pkg/certificates/config.go

```diff
@@ -85,7 +85,7 @@ func (c CertServerConfig) NewAuthDB() (*AuthorizationDB, error) {
 }
 
 // Run implements the responsibility interface, starting a certificate signing server.
-func (c CertServerConfig) Run(ctx context.Context, server *server.Server) (err error) {
+func (c CertServerConfig) Run(ctx context.Context, srv *server.Server) (err error) {
```

Comment:
Let's avoid cleanups in other packages that you are not directly touching.

Reply:
going forward I'll keep an eye out to avoid that :+1: